pom.xml

```diff
@@ -12,7 +12,8 @@
 	</properties>
 
 	<modules>
-		<module>editorconfig-eclipse-plugin</module>
+		<module>org.eclipse.editorconfig.core</module>
```

Comment:
The convention name of Eclipse plugins (that I have seen) follow the convention that I have done. If you wish to migrate to Eclipse.org, I think we should keep this name.

@mickaelistria could you give us your comment please.